### DISCUSSION TITLE
Re-create Azure tags in existing azure environment

### DISCUSSION THREAD

**Discussion by gbalajicl** [microsoft/NubesGen#443] [Q&A]
Hello All,

We are assigning tags to Azure resources.  Some of the tags are assigned using Azure policy, and some of the tags are assigned manually.  
We have assigned tags to resources by using the Azure policy effect "append".
Now we want to assign the same tags with the Policy effect "modify". 

Can you please suggest to me the best process for how to **re-create** tags using IaC Terraform? or **How to re-create** tags using policy? I need suggestions on this tagging **process**. What's the best way to do it and how to do it?

Example tags: ApplicationName, Owner, CostCenter,

**Comment by jdubois** [comment]
For NubesGen we are only creating tags related to the project (like the NubesGen version), and we are mostly targeting new environments.
What we do is therefore very simple, something like you can find in the article here https://build5nines.com/terraform-azure-resource-tags-tips/
It is possible to also create policies (I've found this article: https://medium.com/@kunal.parkar886/using-terraform-and-azure-policies-manage-tag-governance-106bf61c7317 ) but that's far beyond our scope. I'm also not sure if it's better to create/manage the tags using a policy, when we also have Terraform.
